cmd/cnab-to-oci/fixup.go

```diff
@@ -59,7 +59,18 @@ func runFixup(opts fixupOptions) error {
 	if err != nil {
 		return err
 	}
-	if err := remotes.FixupBundle(context.Background(), &b, ref, resolver); err != nil {
+	eventsCh := make(chan remotes.FixupEvent)
+	eventsShown := make(chan struct{})
+	go func() {
+		defer close(eventsShown)
+		for ev := range eventsCh {
+			fmt.Fprintf(os.Stderr, "%s: source=%s, dest=%s, descriptor: %#v, message: %s, error: %s\n", ev.EventType, ev.SourceImage, ev.DestinationRef, ev.Descriptor, ev.Message, ev.Error)
```

Comment:
~~Is this not logging `ev.DigestTree` on purpose?~~ _edit:_ I discovered subsequently how complicated that field is...

This is also mostly dup'd with `push` etc, a helper `displayEvents` might be useful?

Reply:
Yes. I am focusing on consuming from docker/app for now. I'll do the polishing next :)